akka-projection-testkit/src/main/scala/akka/projection/testkit/TestProjection.scala

```diff
@@ -61,11 +62,19 @@ class TestProjection[Offset, Envelope](
       groupAfterEnvelopes: Int,
       groupAfterDuration: FiniteDuration): TestProjection[Offset, Envelope] = this
 
+  /**
+   * INTERNAL API
+   */
+  @InternalApi
   private[akka] def actorHandlerInit[T]: Option[ActorHandlerInit[T]] = None
```

Comment:
we can probably make it `private[projection]`

Reply:
I'd rather review this kind of things on a separate PR